src/Data/String/Base.agda

```diff
@@ -7,6 +7,9 @@ open import Agda.Builtin.String public
     ( primStringUncons   to uncons
     ; primStringToList   to string→list
     ; primStringFromList to list→string
-    ; primStringAppend   to concat-str
```

Comment:
Just rename `primStringAppend` to `_++ₛ_`

Reply:
> Just rename `primStringAppend` to `_++ₛ_`

That was my first try, but it then warns for set fixity for an operator defined in another module.

Reply:
Ah okay then!

Reply:
I was mistaken, see the [manual](https://agda.readthedocs.io/en/v2.6.3.20230805/language/mixfix-operators.html)
>Fixities can be changed when importing with a renaming directive:
```
open M using (_∙_)
open M renaming (_∙_ to infixl 10 _*_)
```